tests/_support/Page/Acceptance/Administrator/ControlPanelPage.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Page\Acceptance\Administrator;
+
+use Page\Acceptance\Administrator\AdminPage;
+
+class ControlPanelPage extends AdminPage
+{
+    public static $url = "/administrator/index.php";
+
+    public static $pageTitle = 'Control Panel';
+
+    public static $pageTitleContext = ['class' => 'page-title'];
```

Comment:
Hmm you are right, you would need to Ctrl+click on Article (or ControlPanelPage in your sample). And then look at the array (or search with Ctrl+F for articleTitleField).

PhpStorm / Shortcut trick (almost as fast as Ctrl+Click)

Shortcut lovers, can do the following: Highlight / mark articleTitleField and do `Cmd + Alt + O (on Mac)`, `Ctrl + Alt + N `(on Linux). This opens search for symbol and is going to show you the variable directly.

<img width="1920" alt="screenshot 2016-06-30 15 58 31" src="https://cloud.githubusercontent.com/assets/897638/16490810/8acbc7d0-3edb-11e6-93b8-5bdfc5a03bee.png">